lib/browser/api/web-contents.js

```diff
@@ -243,14 +243,17 @@ WebContents.prototype.getZoomLevel = function (callback) {
   })
 }
 
-WebContents.prototype.loadFile = function (filePath) {
+WebContents.prototype.loadFile = function (filePath, options = {}) {
```

Comment:
Maybe it would be better to destructure `options` here?
I would serve as documentation for the expected `options` contents:

```js
WebContents.prototype.loadFile = function (filePath, {hash, query, search} = {}) {
  <...>
   return this.loadURL(url.format({
     protocol: 'file',
     slashes: true,
     pathname: path.resolve(app.getAppPath(), filePath),
     query,
     search,
     hash
   }))	
}
```